setup.cfg

```diff
@@ -30,3 +33,10 @@ exclude =
     bin
     build
     venv
+
+rst-roles =
+    # Python programming language:
+    py:func,py:mod,mod,class,attr
+
+rst-directives =
+    autosummary,currentmodule 
```

Comment:
what are these doing?

Reply:
No idea, this is from merging in the most recent version of the plugin from their repo. It's not our changes.